pythonExecSource/src/test/python/test_PyExecConnectorLive.py

```diff
@@ -231,6 +327,51 @@ async def setup_test_env(self, client: Vantiq, use_conn_info: bool):
             assert vr.is_success
             assert vr.errors is None or vr.errors == []
 
+            # Set up procedures to verify compile error processing
+            for k, v in {TEST_SYNTAX_ERROR: py_code_syntax_error,
+                         TEST_INDENT_ERROR: py_code_indent_error,
+                         TEST_IMPORT_ERROR: py_code_import_error}.items():
+                proc_def = {'ruleText': f'''PROCEDURE {k}()
+                var pythonCode = "{v}"
+    
+                var result = select * from source {PY_EXEC_SOURCE_NAME}
+                    with name = "py_code_timeout", code = pythonCode, presetValues = {{ loop_count: 3 }},
+                        codeHandlesReturn = true
+                log.info("Got result: {{}}", [result])
+                return result
+                '''}
+                vr = await client.insert('system.procedures', proc_def)
+                assert vr.is_success
+                assert vr.errors is None or vr.errors == []
+
+            proc_def = {'ruleText': f'''PROCEDURE {TEST_TIMEOUT}()
+            var pythonCode = "{py_code_timeout}"
+
+            try {{
```

Comment:
Note that all the double {{'s and }}'s are to escape the squirrelly brackets when used in a Python f-string (similar to a groovy G-String)